subsystem/intake.py

```diff
@@ -46,3 +47,19 @@ def detect_coral(self) -> bool:
         """
         self.coral_in_intake = self.sensor.getVoltage()
         return self.coral_in_intake
+    
+    def get_current(self) -> int:
```

Comment:
This is going to be an int? Probably not. Can't we get this from the motor somehow? Same for AppliedOutput.